pkg/apis/eventing/v1alpha1/subscription_types.go

```diff
@@ -77,8 +77,6 @@ type SubscriptionSpec struct {
 	//   - Kind
 	//   - APIVersion
 	//   - Name
-	// Kind must be "Channel" and APIVersion must be
```

Comment:
Instead of removing these, we should probably point to saying that the resource must meet the subscribable contract with a pointer to duck type?

Reply:
Also might want to say words about if it turns out not to be, this will be reflected in the status fields, etc. etc etc.

Reply:
Done! Added a similar description to the spec.reply.channel but stating it should be addressable there. This should also fix https://github.com/knative/eventing/issues/925